solana-programs/reward-manager/program/tests/transfer.rs

```diff
@@ -1,23 +1,13 @@
 #![cfg(feature = "test-bpf")]
-mod assert;
 mod utils;
 
-use audius_reward_manager::{
-    instruction,
-    processor::{SENDER_SEED_PREFIX, TRANSFER_ACC_SPACE, TRANSFER_SEED_PREFIX, VERIFY_TRANSFER_SEED_PREFIX},
-    utils::{find_derived_pair, EthereumAddress},
-    vote_message,
-};
+use audius_reward_manager::{error::AudiusProgramError, instruction, processor::{SENDER_SEED_PREFIX, TRANSFER_ACC_SPACE, TRANSFER_SEED_PREFIX, VERIFY_TRANSFER_SEED_PREFIX}, state::{VerifiedMessages}, utils::{find_derived_pair, EthereumAddress}, vote_message};
```

Comment:
nit, probs better on individual lines imo

Reply:
or just import audius_reward_manager::*

Reply:
let's just let the autoformatter deal with it later